packages/roosterjs-editor-dom/lib/style/getBiggestZIndex.ts

```diff
@@ -0,0 +1,22 @@
+import getTagOfNode from '../utils/getTagOfNode';
+
+/**
+ * Search through from element to it's root for the biggest z-index value
+ * @param element the parent element
+ * @returns
+ */
+export default function getBiggestZIndex(element: HTMLElement) {
+    let zIndex = element.style.zIndex ? parseInt(element.style.zIndex) : 0;
+    let child: HTMLElement | null = element;
+    while (child && getTagOfNode(child) !== 'BODY') {
+        const parent: HTMLElement | null = child?.parentElement;
+        if (parent) {
+            const parentZIndex = parent.style?.zIndex ? parseInt(parent.style.zIndex) : 0;
+            if (parentZIndex > zIndex) {
```

Comment:
no, what we need is not the largest z index, but the last z index. e.g.
```html
<body>
 <div zIndex = 1>
   <div zIndex = 100>
    ...
```
Although 1 < 100, but since 1 is the last zindex, we use it instead of 100